src/scripts/convert_lovd2.php

```diff
@@ -0,0 +1,1346 @@
+    'Variant/DNA_published' =>          array('vot',        'VariantOnTranscript/Published_as'),
+    'Variant/DBID' =>                   array('vog',        'VariantOnGenome/DBID',         'lovd_convertDBID'),
+    'Variant/Restriction_site' =>       array('vog',        'VariantOnGenome/Restriction_site'),
+    'Variant/Remarks' =>                array('vog',        'VariantOnGenome/Remarks'),
+    'Variant/Genetic_origin' =>         array('vog',        'VariantOnGenome/Genetic_origin'),
+    'Variant/Detection/Technique' =>    array('screening',  'Screening/Technique',          'lovd_convertScrTech'),
+    'Variant/Reference' =>              array('vot',        'VariantOnTranscript/Reference','lovd_convertReference'),
+    'Patient/Patient_ID' =>             array('individual', 'Individual/Lab_ID'),
+    'Patient/Reference' =>              array('individual', 'Individual/Reference',         'lovd_convertReference'),
+    'Patient/Gender' =>                 array('individual', 'Individual/Gender',            'lovd_convertGender'),
+    'Patient/Times_Reported' =>         array('individual', 'panel_size'),
+    'Patient/Phenotype_2' =>            array('phenotype',  'Phenotype/Additional'),
+    // Note that 'Patient/Phenotype/Inheritance' automatically maps to Phenotype/Inheritance as well.
+    'Patient/Occurrence' =>             array('phenotype',  'Phenotype/Inheritance',        'lovd_convertInheritance'),
+    'Patient/Mutation/Origin' =>        array('vog',        'VariantOnGenome/Genetic_origin',   'lovd_convertOrigin'),
+    'ID_pathogenic_' =>                 array('vog',        'effectid'),
+    'ID_status_' =>                     array('vog',        'statusid',                     'lovd_convertStatus'),
+    'ID_variant_created_by_' =>         array('vog',        'created_by',                   'lovd_convertCuratorID'),
+    'variant_created_date_' =>          array('vog',        'created_date'),
+    'ID_variant_edited_by_' =>          array('vog',        'edited_by',                    'lovd_convertCuratorID'),
+    'variant_edited_date_' =>           array('vog',        'edited_date'),
+    'ID_patient_created_by_' =>         array('individual', 'created_by',                   'lovd_convertCuratorID'),
+    'patient_created_date_' =>          array('individual', 'created_date'),
+    'ID_patient_edited_by_' =>          array('individual', 'edited_by',                    'lovd_convertCuratorID'),
+    'patient_edited_date_' =>           array('individual', 'edited_date'),
+    'ID_patientid_' =>                  array('individual', 'id',                           'lovd_autoIncIndividualID'),
+    'ID_variantid_' =>                  array('vog',        'id',                           'lovd_autoIncVariantID'),
+    'ID_allele_' =>                     array('vog',        'allele'),
+    'ID_submitterid_' =>                array('vog',        'owned_by',                     'lovd_convertSubmitterID'),
```

Comment:
Just agreed to these changes:
- The `owned_by` field should not be filled in from the static curator ID directly, only perhaps indirectly (see below).
- The `owned_by` field should contain the translated `ID_submitterid_` field, the static submitter ID, or otherwise the *translated* value that is in the `created_by` field. So it's best to apply the static curator ID first to the `created_by`, before you fill the `owned_by` field.